README.md

```diff
@@ -235,13 +232,13 @@ Having the Infinispan Operator deployed, just jump to [Install Data Index Servic
 
 ### Deploy Strimzi
 
-Deploying [Strimzi](https://strimzi.io/) is easy since it's an Operator and should be available in the [OperatorHub](https://operatorhub.io/operator/strimzi-kafka-operator). On OpenShift Web Console, go to the left menu, Catalog, OperatorHub and search for `Strimzi`.
+Find the [Strimzi](https://strimzi.io/) Operator in the [OperatorHub](https://operatorhub.io/operator/strimzi-kafka-operator). On OpenShift Web Console, go to the left menu, `Operators`, `Installed Operators` and search for `Strimzi`.
 
-Follow the on screen instructions to install the Strimzi Operator. At the end, you should see the Strimzi Operator on the Installed Operators tab:
+You should see the Strimzi Operator on the Installed Operators tab:
 
 ![Strimzi Installed](docs/img/strimzi_installed.png?raw=true)
 
-Next, you need to create a Kafka cluster and a Kafka Topic for the Data Index Service to connect. Click on the name of the Strimzi Operator, then on `Kafka` tab and `Create Kafka`. Accept the default options to create a 3 node Kafka cluster. If it's a development environment, consider setting the Zookeeper and Kafka replicas to 1 to save resources.
+Next, you need to create a Kafka cluster for the Data Index Service to connect. Click on the name of the Strimzi Operator, then on `Kafka` tab and `Create Kafka`. Accept the default options to create a 3 node Kafka cluster. If it's a development environment, consider setting the Zookeeper and Kafka replicas to 1 to save resources.
```

Comment:
Couldn't we do it via Kogito Operator ? Or is there any blocker ?
I mean we are already creating the Kafka Topics automatically and we are able to find the instance automatically if no provided (see cmd/kogito/command/install/data-index.go)

Reply:
This should be handled by the Kogito Operator like is handled with Infinispan. A Kafka cluster should be created like Infinispan are with `KogitoInfra` controller.

Reply:
We will do this in https://issues.jboss.org/browse/KOGITO-614 .